src/handlers/RequestHandler.ts

```diff
@@ -272,3 +285,22 @@ export abstract class RequestHandler<
     }
   }
 }
+
+/**
+ * Bypass this intercepted request.
+ * This will make a call to the actual endpoint requested.
+ */
+export function passthrough(): MockedResponse<null> {
+  // Constructing a dummy "101 Continue" mocked response
+  // to keep the return type of the resolver consistent.
+  return {
```

Comment:
I'm returning a `MockedResponse` from `passthrough` to keep the same return type of the response resolver. I don't like this approach but I'll use it for now. 

We need to refactor the return type of the resolver to be some sort of `Instruction`, which could be:

- Passthrough
- Fallthrough
- Use a mocked response
- Return a network error
- Anything else in the future

It should be a strictly defined object representing the library's next steps. Resolvers don't necessarily return mocked responses, as in the Passthrough/Fallthrough scenarios. Each intention can also be permanent or one-time (like `res.once()`). This is a great opportunity to solve #413.

Reply:
Proposing this in https://github.com/mswjs/msw/issues/1207.